CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Features:
 	- Added new API method to close displayed tooltips [SDKCF-6027]
 	- Added contexts validation for tooltip campaigns [SDKCF-6028]
+	- Added tracking of opt-in/opt-outs in Push Primer feature [SDKCF-6026]
```

Comment:
Please add a section or note on how to use the new API.

Reply:
but it's not a new API. We don't have anything about analytics events sent by the SDK